types/browserslist/index.d.ts

```diff
@@ -1,15 +1,27 @@
-// Type definitions for browserslist 4.4
+// Type definitions for browserslist 4.8
 // Project: https://github.com/browserslist/browserslist#readme
 // Definitions by: Dave Cardwell <https://github.com/davecardwell>
 //                 Andrew Leedham <https://github.com/AndrewLeedham>
+//                 Piotr Błażejewicz <https://github.com/peterblazejewicz>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.7
 
 declare namespace browserslist {
+    /** statistics from `Browserslist` files */
+    type MyStats = 'my stats';
```

Comment:
`"my stats" | string` is equivalent to `string` from the compiler's point of view, but the additional type alias can be useful for documentation.